pkgs/xdg-desktop-portal-wlr/default.nix

```diff
@@ -18,11 +16,16 @@ stdenv.mkDerivation rec {
     sha256 = metadata.sha256;
   };
 
-  nativeBuildInputs = [ pkgconfig meson ninja ];
-  buildInputs = [ systemd wayland wayland-protocols pipewire libdrm ];
-  mesonFlags = [ "-Dauto_features=auto" ];
+  nativeBuildInputs = [ meson ninja pkg-config wayland-protocols makeWrapper ];
+  buildInputs = [ pipewire wayland systemd libdrm iniparser scdoc inih ];
 
-  enableParallelBuilding = true;
+  mesonFlags = [
+    "-Dsd-bus-provider=libsystemd"
+  ];
+
+  postInstall = ''
+    wrapProgram $out/libexec/xdg-desktop-portal-wlr --prefix PATH ":" ${lib.makeBinPath [ grim slurp ]}
+  '';
```

Comment:
And I suppose this will enable the use of the new display "chooser" added upstream?

Reply:
yes slurp is the selector